config/forms/2021_2022.json

```diff
@@ -1111,23 +1123,44 @@
                   "hint_text": "",
                   "type": "text"
                 }
-              }
+              },
+              "depends_on": [{ "startertenancy": "Yes" }]
             },
-            "letting_provider": {
+            "fixed_term_tenancy": {
               "header": "",
               "description": "",
               "questions": {
-                "landlord": {
```

Comment:
@dushan-madetech  @paulrobertlloyd  it looks like `landlord` dropped out of the form here without being replaced by a derived variable but it is in the data dictionary. That doesn't seem correct? Should it be derived?